docs/usage.rst

```diff
@@ -34,9 +34,27 @@ Non-blocking lock::
     else:
         print("Someone else has the lock.")
 
-Controlled execution lock::
+Releasing previously acquired lock::
+
+    conn = StrictRedis()
+    lock = redis_lock.Lock(conn, "name-of-the-lock")
+    lock.acquire()
+    print("Got the lock. Doing some work ...")
+    time.sleep(5)
+    lock.release()
+
+The above example could be rewritten using context manager::
 
     conn = StrictRedis()
     with redis_lock.Lock(conn, "name-of-the-lock"):
         print("Got the lock. Doing some work ...")
         time.sleep(5)
+
+If lock was acquired in other context (other process, remote worker server,
+etc), it's still possible to release it with parameter `force` equal
+to ``True``::
```

Comment:
Now we could fix `reset` but then what about `reset_all`? LPUSHisng tons of keys might be too slow. Still, would be the right thing.

In hindsight, had I named this argument `error` or `strict` instead `force` it would had been more clearer to its purpose :(